org.freedownload.Manager.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: org.freedownload.Manager
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: .
+command: fdm
+tags:
+  - proprietary
+finish-args:
+  - --socket=wayland 
+  - --socket=x11
```

Comment:
if it supports wayland then:

```suggestion
  - --socket=wayland 
  - --socket=fallback-x11
```

otherwise remove the wayland socket.

Reply:
![image](https://user-images.githubusercontent.com/84112405/172279212-e7a4b13c-1c34-4b72-93bc-ca5e8f1831b0.png)

Do you suggest that it should be removed and only allowed to run on x11? I understand that all apps that start using qt6 will get wayland support, but it seems that you are still running the LTS version of qt (5.15)

Reply:
> but it seems that you are still running the LTS version of qt (5.15)

I this ship qt6 then qt6 is run. Nevermind my suggestion. It still seem that Qt wayland support is subpar (I'm being nice) and they continue to not want to fix it by ignoring Wayland when run with GNOME. So the easiest is to leave it at that.
(the Qt version in the runtime is patched)